Sources/JavaScriptKit/JSValue.swift

```diff
@@ -57,12 +57,21 @@ public enum JSValue: Equatable {
 
     /// Returns the `true` if this JS value is null.
     /// If not, returns `false`.
-    public var isNull: Bool { return self == .null }
+    public var isNull: Bool {
+        return self == .null
+    }
 
     /// Returns the `true` if this JS value is undefined.
     /// If not, returns `false`.
-    public var isUndefined: Bool { return self == .undefined }
+    public var isUndefined: Bool {
+        return self == .undefined
+    }
+}
```

Comment:
Now that I’m reviewing this section — how are these useful? Isn’t it better to use one of the other type accessors?

It looks like they’re used in `JSValueDecoder` and nowhere else.

Reply:
I concur, but Yuta should make the call 🙂

Reply:
I'm thinking changing JSValue to be struct instead enum to be more resilient interface. And for future coming JSString implementation. 

If changing from enum to struct, value == .null will be broken. So I may restore this in the future. But for now, it's ok to remove them.

Reply:
If you add `static let null = ...` to your new JSValue implementation, `value == .null` will still work!